.github/CODEOWNERS

```diff
@@ -70,3 +70,6 @@
 
 /apps/skilavottord/                                       @island-is/deloitte
 /libs/skilavottord/                                       @island-is/deloitte
+
+# /libs/auth-api-lib/                                     @island-is/fuglar
```

Comment:
Why is it commented?

Reply:
I got weird bugs in the build that the team had no permission or something. I will uncomment this in the next PR when the new lib is part of main